adagrams/game.py

```diff
@@ -1,11 +1,86 @@
-def draw_letters():
-    pass
+import random
+from typing import Counter
+letter_bank = {
```

Comment:
Good job defining this large data object outside the functions. This lets the functions just have the logic, rather than being overshadowed by data definitions. Also, since we're treating this dictionary as a constant, we can use the upper-case letters to indicate it, i.e. `LETTER_BANK`